synapse/models/orgs.py

```diff
@@ -734,6 +744,41 @@ def getModelDefs(self):
                     ('org', ('ou:org', {}), {}),
                     ('name', ('ou:name', {}), {}),
                 )),
+
+                ('ou:asset', {}, (
```

Comment:
Do we want to add a :first / :start and :last / :end date props to capture when the node first became an asset / was last considered an asset?